src/solvers/miniBDD/miniBDD.cpp

```diff
@@ -22,6 +22,7 @@ Author: Daniel Kroening, kroening@kroening.com
 
 void mini_bdd_nodet::remove_reference()
 {
+  // NOLINTNEXTLINE(build/deprecated)
```

Comment:
Use `PRECONDITION()`, `INVARIANT()`, etc  from https://github.com/diffblue/cbmc/blob/master/src/util/invariant.h rather than silencing the linter.